packages/web/hooks/ui-config/use-lock-token-config.ts

```diff
@@ -115,6 +115,7 @@ export function useLockTokenConfig(sendCurrency?: AppCurrency | undefined): {
       queryOsmosis,
       queryOsmosis.querySyntheticLockupsByLockId,
       queryOsmosis.queryLockableDurations.response,
+      account?.osmosis,
```

Comment:
> Nice, it used to not change, but now with multiple wallets the account object reference will change now, right?

With Mobx we didn't have to worry about this. Unfortunately, we can't convert this object since it's nested and coming from Cosmoskit. 

> May want to do a CMD+F and make sure this is not happening elsewhere.

Yeah, I didn't find any more missing references.